cloud-controller-manager/do/loadbalancers.go

```diff
@@ -833,6 +844,13 @@ func buildForwardingRules(service *v1.Service) ([]godo.ForwardingRule, error) {
 			protocol = protocolHTTP2
 		}
 
+		if port.Protocol == v1.ProtocolUDP {
+			if tcpPortMap[port.Port] {
+				return nil, errors.New("cannot share a port between TCP and UDP")
```

Comment:
Can we include the shared port in the returned error message?

Reply:
Added!

Reply:
Added!